spot-client/src/common/remote-control/BaseRemoteControlService.js

```diff
@@ -105,8 +105,7 @@ export class BaseRemoteControlService extends Emitter {
     }
 
     /**
-     * Returns the current join code that is necessary to establish a connection
-     * to a Spot-TV.
+     * Returns the current join code that is necessary to establish a connection with the service.
```

Comment:
Let me know when you I can go through the docs (including the readme files) and I'll update everything as needed. Sometimes it's nice product something without having to think really hard.